src/connectivity.jl

```diff
@@ -178,6 +178,9 @@ true
 """
 is_weakly_connected(g) = is_connected(g)
 
+
+
+
```

Comment:
I don't think we need all this whitespace here.

Reply:
Removed.